_data/collection-index.yml

```
@@ -822,4 +822,9 @@
   maintainer: Carsten Behring
   contact: https://github.com/scicloj/devcontainer-templates/issues
   repository: https://github.com/scicloj/devcontainer-templates/
-  ociReference: ghcr.io/scicloj/devcontainer-templates  
+  ociReference: ghcr.io/scicloj/devcontainer-templates
+- name: Synology devcontainer templates
+  maintainer: ChaosWars
+  contact: https://github.com/ChaosWars/synology-features/issues
+  repository: https://github.com/ChaosWars/synology-features/
+  ociReference: ghcr.io/ChaosWars/synology-features
```

```suggestion
  ociReference: ghcr.io/chaoswars/synology-features
```

Hi 👋

The ORAS spec requires the `ociReference` to be in lower case. Can we update here, thanks?

The supporting tools do help convert the case, however, its better to sync as much as possible.